test/torchaudio_unittest/common_utils/case_utils.py

```diff
@@ -253,6 +253,16 @@ def skipIfNoModule(module, display_name=None):
     ),
     key="ON_PYTHON_310",
 )
+skipIfNoDevice = _skipIf(
```

Comment:
Can we rename this to `skipIfNoAudioDevice`?
In PyTorch context, normally device refers to compute device like CUDA